app/code/Magento/Theme/Model/Design/Config/Storage.php

```diff
@@ -87,10 +87,14 @@ public function load($scope, $scopeId)
                 $scopeId,
                 $fieldData->getFieldConfig()
             );
-            if ($value !== null) {
-                $fieldData->setValue($value);
+
+            if ($value === null) {
+                $value = '';
+
```

Comment:
Remove blank line at end of control structure

Reply:
@sidolov, done and squashed.